gapis/api/vulkan/vulkan.go

```diff
@@ -291,7 +307,41 @@ func (API) ResolveSynchronization(ctx context.Context, d *sync.Data, c *path.Cap
 			case VkCmdDebugMarkerEndEXTArgsʳ:
 				popMarker(DebugMarker, uint64(i), nCommands)
 			}
+
+			// Handle draw commands grouping.
+			cmdName := ""
+			if generatingId != nil {
```

Comment:
When this fails (recording was prior to MEC start) we're going to have a problem.

Reply:
Thanks for pointing it out! The command name is no longer referenced from the generatingId after the new cimmits.